support-frontend/assets/stylesheets/skeleton/non-critical-webfonts.scss

```diff
@@ -0,0 +1,33 @@
+// Our core fonts (Guardian Headline, Guardian Text Egyptian Web and Guardian Text Sans Egyptian Web)
+// are loaded via an iFrame from dotcom. If you use any fonts in this file, it will result in
+// the font files being downloaded from pasteup
```

Comment:
this isn't really the problem that they will be downloaded from pasteup, the problem is that will actually slow everything else down due to bandwidth contention with the critical stuff.
Being downloaded from pasteup would be okay if that was triggered by the JS after the critical stuff was in memory.

Reply:
I'm not saying it's a problem, I'm just saying what will literally happen

Reply:
I guess I could say "unless they are already in the browser cache"

Reply:
although IMO thats an broswer implentation detail